testing/trino-product-tests/src/main/java/io/trino/tests/utils/SchemaRegistryClientUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.tests.utils;
+
+import io.confluent.kafka.schemaregistry.client.CachedSchemaRegistryClient;
+import io.confluent.kafka.schemaregistry.client.SchemaRegistryClient;
+import io.trino.tempto.configuration.Configuration;
+
+import java.util.Map;
+
+import static com.google.common.base.Preconditions.checkArgument;
+import static io.trino.tempto.context.ThreadLocalTestContextHolder.testContext;
+
+public class SchemaRegistryClientUtils
+{
+    private SchemaRegistryClientUtils()
+    {
+    }
+
+    public static SchemaRegistryClient getSchemaRegistryClient()
+    {
+        Map<String, Object> schemaRegistryProperties = testContext().getDependency(Configuration.class)
+                .getSubconfiguration("databases.kafka.schema-registry").asMap();
+
+        checkArgument(schemaRegistryProperties.containsKey("url"), "schemaRegistry url is missing");
+
+        return new CachedSchemaRegistryClient(
+                (String) schemaRegistryProperties.get("url"),
+                100,
```

Comment:
what is 100? Maybe extract a variable that is named exactly as constructor parameter?